File: 프로그래머스 코테(java)/프로그래머스0(1~30번)/8.홀짝 구분하기.java
// 레벨 0
// https://school.programmers.co.kr/learn/courses/30/lessons/181944?language=java

import java.util.Scanner;

class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        if (n % 2 == 0) {
            System.out.printf("%d is even", n);
        } else {
            System.out.printf("%d is odd", n);
        }
        // System.out.print(n + " is "+(n % 2 == 0 ? "even" : "odd")); 이렇게 쓴사람도 있음
    }
}

File: 프로그래머스 코테(js)/프로그래머스0(1~30번)/8.홀짝 구분하기.js
// 레벨 0
// https://school.programmers.co.kr/learn/courses/30/lessons/181944?language=java

const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let input = [];

rl.on('line', function (line) {
    input = line.split(' ');
}).on('close', function () {
    const n = Number(input[0]);
    
    let answer = '';
    if (n % 2 === 0) { // 짝수일 경우
        answer = n + " is even"
    } else { // 홀수일 경우
        answer = n + " is odd"
    }
    
    console.log(answer);
});
